core_impl/src/test/java/com/google/instrumentation/trace/TraceParamsTest.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2017, Google Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.instrumentation.trace;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Unit tests for {@link TraceParams}. */
+@RunWith(JUnit4.class)
+public class TraceParamsTest {
+
+  @Test
+  public void defaultTraceParams() {
+    assertThat(TraceParams.DEFAULT.getSampler()).isEqualTo(TraceParams.DEFAULT_SAMPLERS);
+    assertThat(TraceParams.DEFAULT.getMaxNumberOfAttributes())
+        .isEqualTo(TraceParams.DEFAULT_SPAN_MAX_NUM_ATTRIBUTES);
+    assertThat(TraceParams.DEFAULT.getMaxNumberOfAnnotations())
+        .isEqualTo(TraceParams.DEFAULT_SPAN_MAX_NUM_ANNOTATIONS);
+    assertThat(TraceParams.DEFAULT.getMaxNumberOfNetworkEvents())
+        .isEqualTo(TraceParams.DEFAULT_SPAN_MAX_NUM_NETWORK_EVENTS);
+    assertThat(TraceParams.DEFAULT.getMaxNumberOfLinks())
+        .isEqualTo(TraceParams.DEFAULT_SPAN_MAX_NUM_LINKS);
```

Comment:
I think it would it be better to also test the constants' values in this test method, for example, `assertThat(TraceParams.DEFAULT.getMaxNumberOfLinks()).isEqualTo(128);`.